pkg/pillar/cmd/zedagent/handleconfig.go

```diff
@@ -356,9 +358,9 @@ func configTimerTask(getconfigCtx *getconfigContext, handleChannel chan interfac
 	iteration := 0
 	withNetTracing := traceNextConfigReq(ctx)
 	retVal, tracedReqs := getLatestConfig(getconfigCtx, iteration, withNetTracing)
-	configProcessingSkipFlag := retVal == skipConfig
-	if configProcessingSkipFlag != getconfigCtx.configProcessingSkipFlag {
-		getconfigCtx.configProcessingSkipFlag = configProcessingSkipFlag
+	configProcessingSkipFlagReboot := retVal == skipConfigReboot
+	if configProcessingSkipFlagReboot != getconfigCtx.configProcessingSkipFlagReboot {
+		getconfigCtx.configProcessingSkipFlagReboot = configProcessingSkipFlagReboot
```

Comment:
See below as well, but it makes sense to save all of retVal in getconfigCtx.

Reply:
Fixed